### PR TITLE
add details about avoiding loadbalanced and none loadbalanced dnspolicy

### PR DESCRIPTION
**Pull request by maleck13**
Signed-off-by: craig <cbrookes@redhat.com>

rh-pre-commit.version: 2.2.0
rh-pre-commit.check-secrets: ENABLED

### CODE REVIEW

File: doc/user-guides/dnspolicy/load-balanced-dns.md [hunk 1]
```diff
@@ -80,3 +80,109 @@ To see all regions supported by GCP Cloud DNS, please see the official (document
 
 To see the different values you can use for the geo based DNS with Azure take a look at the following (documentation)[https://learn.microsoft.com/en-us/azure/traffic-manager/traffic-manager-geographic-regions]
 
+### Moving from none load balanced to load balanced or visa versa
+
+It is possible to update a DNSPolicy that has no load balancing options set to one that has these options set and visa versa. Underneath, the DNS Operator will remove the existing records and replace them with the correct set of records based on your configuration. It is important however that when using DNSPolicy across multiple Gateways that share a hostname, the DNSPolicies targeting a listener with a shared hostname all use a load balancing configuration or all don't use load balancing configuration (IPAddress only). It is invalid to have two DNSPolcies targeting a listener with a shared hostname that use different dns `strategies`. Doing so will cause one of the DNSPolicies to fail to be enforced and report an error caused by an inability to bring the DNS records into a consistent state.
```
Comment (philbrookes) [inline doc/user-guides/dnspolicy/load-balanced-dns.md:85]: vice versa

Comment (philbrookes) [inline doc/user-guides/dnspolicy/load-balanced-dns.md:85]: > It is important however that when using DNSPolicy across multiple Gateways that share a hostname, the DNSPolicies targeting a listener with a shared hostname all use a load balancing configuration or all don't use load balancing configuration (IPAddress only).

It is important that all of the DNS Policies that target a listener with the same hostname all use the same load balancing configuration (or absence thereof).

File: doc/user-guides/dnspolicy/load-balanced-dns.md [hunk 2]
```diff
@@ -80,3 +80,109 @@ To see all regions supported by GCP Cloud DNS, please see the official (document
 
 To see the different values you can use for the geo based DNS with Azure take a look at the following (documentation)[https://learn.microsoft.com/en-us/azure/traffic-manager/traffic-manager-geographic-regions]
 
+### Moving from none load balanced to load balanced or visa versa
```
Comment (mikenairn) [inline doc/user-guides/dnspolicy/load-balanced-dns.md:83]:
```suggestion
### Moving from non load balanced to load balanced or visa versa
```

File: doc/user-guides/dnspolicy/load-balanced-dns.md [hunk 3]
```diff
@@ -80,3 +80,109 @@ To see all regions supported by GCP Cloud DNS, please see the official (document
 
 To see the different values you can use for the geo based DNS with Azure take a look at the following (documentation)[https://learn.microsoft.com/en-us/azure/traffic-manager/traffic-manager-geographic-regions]
 
+### Moving from non load balanced to load balanced or vice versa
+
+It is possible to update a DNSPolicy that has no load balancing options set to one that has these options set and visa versa. Underneath, the DNS Operator will remove the existing records and replace them with the correct set of records based on your configuration. It is important however that when using DNSPolicy across multiple Gateways that share a hostname, the DNSPolicies targeting a listener with a shared hostname all use a load balancing configuration (or absence thereof). It is invalid to have two DNSPolcies targeting a listener with a shared hostname that use different dns `strategies`. Doing so will cause one of the DNSPolicies to fail to be enforced and report an error caused by an inability to bring the DNS records into a consistent state.
```
Comment (philbrookes) [inline doc/user-guides/dnspolicy/load-balanced-dns.md:85]: you still have 'visa versa' here, instead of 'vice versa'.

Comment (maleck13) [reply doc/user-guides/dnspolicy/load-balanced-dns.md]: ok fixed

File: doc/user-guides/dnspolicy/load-balanced-dns.md [hunk 4]
```diff
@@ -9,7 +9,7 @@ This document will show you how to setup a load balanced DNS configuration using
 
 It is most useful to use the load balancing options when targeting multiple gateways that share a listener host E.G (api.example.com). It is also perfectly valid to use it when you only have a single gateway; this provides the benefit of allowing you to easily expand beyond this single gateway for a given shared hostname. It is worth knowing that the load balanced DNSpolicy comes with a relatively small additional cost of some added records and lookups during DNS resolution vs a "simple" DNSPolicy with no load balancing specified as the latter only sets up a simple A or CNAME record. So in summary if you expect to need multiple gateways for a given listener host then you should take advantage of the load balanced option.
 
-
+moving-between-simple-loadbalanced
```
Comment (mikenairn) [inline doc/user-guides/dnspolicy/load-balanced-dns.md:12]: Is this intentional?

Comment (maleck13) [reply doc/user-guides/dnspolicy/load-balanced-dns.md:12]: nope... sigh doing too many things at once.

**Review by philbrookes — APPROVED** [review]
LGTM

**Pull request merged**